src/current/v24.2/operational-faqs.md

```diff
@@ -244,7 +244,7 @@ You can also see these metrics in [the Clock Offset graph]({% link {{ page.versi
 
 ## How do I prepare for planned node maintenance?
 
-Perform a [node shutdown]({% link {{ page.version.version }}/node-shutdown.md %}#perform-node-shutdown) to temporarily stop a node that you plan to restart.
+Perform a [node shutdown]({% link {{ page.version.version }}/node-shutdown.md %}#perform-node-shutdown) to temporarily stop the `cockroach` process on a node that you plan to restart.
```

Comment:
```suggestion
Perform a [node shutdown]({% link {{ page.version.version }}/node-shutdown.md %}#perform-node-shutdown) to temporarily stop the `cockroach` process of a node that you plan to restart.
```

Reply:
It should be "on" because the `cockroach` process is running on the node. After you stop the `cockroach` process you can restart your server or do whatever other maintenance you want. You don't have to restart the node. But I think this is trying to say "that you don't plan to decommission from the cluster". Edited the edit.